src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -809,6 +809,8 @@ private function rowToArray(
         }
 
         if ($document instanceof LocalizedAuthorBehavior) {
+            $documentData['isUpdated'] = $document->getLastModifiedEnabled();
+            $documentData['updated'] = $document->getLastModified();
```

Comment:
I think we do not require the Enabled here so we can remove it and rename `updated`

```suggestion
            $documentData['lastModified'] = $document->getLastModified();
```